Sources/UI/UITableView+Diff.swift

```diff
@@ -1,3 +1,4 @@
+import OptiDiff
```

Comment:
This causes issues in CocoaPods, let's try `canImport`:
```suggestion
#if canImport(OptiDiff)
  import OptiDiff
#endif
```

Reply:
Didn't work. Need to find another way of organizing the modules.